test/api/breaks_api_test.rb

```diff
@@ -0,0 +1,28 @@
+require 'test_helper'
+
+class BreaksApiTest < ActiveSupport::TestCase
+include Rack::Test::Methods
+include TestHelpers::AuthHelper
+include TestHelpers::JsonHelper
+
+def app
+    Rails.application
+end
+
+# GET TEST 
+# Get All Breaks
+def test_get_all_breaks
+    user = User.admins.first
+    get with_auth_token('/api/breaks',user)
```

Comment:
Can you check where this is used in the web project? I have a feeling this endpoint is not needed, in which case we should delete the endpoint rather than creating tests for it.

Look for a resource plus entity for the break url.

Reply:
Do you need a user to access this endpoint? Is it authenticated?

Reply:
Hi Andrew,

I've taken a look at the web project and i can't find any use for this endpoint.
According to [this](https://github.com/doubtfire-lms/doubtfire-web/blob/development/src/app/api/models/break.coffee) we access breaks via teaching periods.

Reply:
Great - you can remove the test and delete the endpoint code.

Reply:
@areejasiri please click resolve on this one